src/pages/about.astro

```diff
@@ -16,43 +16,50 @@ import FooterFull from '../components/footer/FooterFull.svelte'
     <img src="/ega.svg" />
   </div>
   <div
-    class="relative max-h-screen overflow-y-scroll font-expressway red-radial-gradient scrollbar-hide snap-start snap-y snap-mandatory"
+    class="relative max-h-screen overflow-y-scroll font-expressway red-radial-gradient scrollbar-hide"
```

Comment:
I think you will probably need to change the horizontal size away from max-h-screen, which was used to break up the page for the snapping. The extra space doesn't line up with the figma.

Reply:
updated and fixed spacing